commec/config/screen_databases.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+# Copyright (c) 2021-2024 International Biosecurity and Biosafety Initiative for Science
+"""
+Defines the `Input and Output Screen Parameters` class, and associated dataclasses.
+"""
+import os
+from commec.config.io_parameters import ScreenIOParameters
+from commec.tools.blastn_handler import BlastNDataBase
+from commec.tools.blastx_handler import BlastXDataBase
+from commec.tools.blastdmnd_handler import DiamondDataBase
+from commec.tools.cmscan_handler import CmscanDataBase
+from commec.tools.hmm_handler import HMMDataBase
+
+class CommecDatabases():
+    """
+    Consolidation and initialisation of all databases needed for the commec workflow.
+    This could be maybe split into a Biorisk_handler, Taxonomy handler, and Benign handler
+    class objects respectively.
+    """
+    def __init__(self, params : ScreenIOParameters):
+        # Biorisk
+        self.biorisk_db : HMMDataBase = None
+        # Taxonomy
+        self.protein_db : BlastXDataBase = None
```

Comment:
Is this going to be an issue, given that we sometimes set this as a `DiamondDatabase`?

Reply:
Typing is just a suggestion in Python, and not enforced - but yes - initially I was just going to make all of these the abstract base class of DatabaseHandler, such that they could be changed however, but then figured using derived classes made sense for the commec specific use case.

This shouldn't cause any bugs or issues, but perhaps we simply change to the base class.

Reply:
removed type suggestion, as its set appropriately further on in __init__